xeno/shields.md

```diff
@@ -0,0 +1,112 @@
+
+A "buff ability" is a replacement for shields, and is the name that is used to refer to any ability that can replace shields.
+
+Clarity stacks are limited effect stacks that let xenos resist minor stuns, can be seen on boxer warrior. These can be used as replacement buffs for shields
+
+Warden is a xeno that can provide, as one of its abilities, short-term buffs or heals. (not long term spam shields)
+
+Shaman is a xeno that can provide, as one of its abilities, short-term AOE buffs or AOE heals. (not long term shields for the entire hive such as in hijack)
+
+Warrior is a xeno that is hard to kill in a 1v1 and takes a frontliner / ambusher role with a preferably higher focus on frontlining. It is not a xeno that can easily stay far away from the main frontline without nearby weeds or assistance like lurker, it is not a xeno that can very easily disengage and run away like lurker. 
+
+## Specific changes (preliminary ideas)
+
+Shields to be removed from all but crusher, rav
+
+Warden shield - replace with a speed boost / warrior clarity stacks / fire extinguish? -> a buff that helps xenos at the front line, and can help them while they are at full health
+
+Carrier shield - replace with aoe clarity stacks / a 10s aoe clarity effect / slash damage -> some other kind of buff that makes xenos better offensively
+
+Vanguard can prob keep shield since it’s just a one free hit mechanic 
+
+Warrior should prob get like 5 extra armour and not get the health nerf from !2501
+
+Base Prae shield from spit also needs to be removed, it’s not super significant so maybe just a tiny health buff, but can prob be just fine with no change
+
+Shield fruit to be replaced with another type of short-term buff fruit (such as speed, damage, clarity, plasma, heal over time)
+
+
+## Alternatives
+
+Good alternatives:
+
+1. Cut down on some of the xeno buffs and replace them with marine nerfs
+
+2. Make the castes with buff abilities get some offensive abilities instead
+
+Bad alternatives:
+
+1. "Just change shields to extra health" - doesn't give them a telegraph, is equally inconsistent and has the same problems as shields
+
+2. "Just fix the problems with shields" - boring, would result in the game being even more arcade-y (shields are already EXTREMELY arcade-y), the goal of rectifying the reasons for this shift to buff abilities can't really be accomplished without it.
+
+3. "Just give shields glows and nerf them slightly" - see above
+
+## Potential Changes
+
+1. Change the type of replacement abilities / buffs that warden + shaman get
+
+2. Let the shield fruit stay
+
+3. Make some marine balance changes, such as an m2c nerf or AP nerf
+
+4. Give additional buffs to xenos to compensate
```

Comment:
Not what this category is supposed to be used for. Here, you list potential problems with your design (e.g. the shield may still cause X problem which can be solved by making Y change)

If you don't know what to put, you can just remove this section

Reply:
done